Week-1/Week.html

```diff
@@ -0,0 +1,142 @@
+<!-- 
+Online HTML, CSS and JavaScript editor to run code online.
+-->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>CattyBoard</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+    <style>
+        #leftbar {
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 50px;
+            height: 100vh;
+            background-color: rgb(190, 188, 188)
+        }
+        #image{
+            margin-left: 100px;
+            height: 200px;
+            width: 200px;
+            margin-top: 30px;
+        }
+        #title1{
+            font-family: 'Lato';
+            margin-top: 60px;
+            margin-left: 20px;
+        }
+        #info{
+            font-family: 'Lato';font-size: 12px;
+        }
+        table,th,tr{
+            font-family: 'Lato';font-size: 18px;
+        }
+        #dark_row{
+            background-color: rgb(168, 168, 168);
+            border-color: rgb(168, 168, 168);
+        }
+        #upper{
+            display: flex;
+            flex-direction: row;
+            background-color: rgb(57, 56, 56);
+            margin-top: 0%; 
+            width:100vw;
+            height: 35vh;
+        }
+        #bottom{
+            display: flex;
+            flex-direction: row;
+            width: 100vw;
+            height: 150px;
+            background-color:rgb(52, 101, 186);
+            position: fixed; bottom: 0; right: 0; left: 0;
+        }
+        #icon{
```

Comment:
Same here - we can target the `icon` tags directly using `icon` instead of having an `id` for it